src/Keboola/StorageApi/Client.php

```diff
@@ -2399,7 +2401,9 @@ public function apiPostMultipart($url, $postData = null, $handleAsyncTask = true
     public function apiPostJson($url, $data = [], bool $handleAsyncTask = true, array $requestOptions = [])
     {
         $requestOptions = $this->filterRequestOptions($requestOptions);
-        $requestOptions['json'] = $data;
+        if ($data !== []) {
```

Comment:
Tady pokud data === null tak to vloží? Vlastně nechápu proč to nemůže poslat prázdný objekt v requestu...

Reply:
jestli jde o `[] vs {}` tak bych to vyřešil, že pokud je $data prázdné pole, tak $data = new stdClass()

Reply:
tohle je lepsi 👍 
upraveno, pokud je pole prazdne, posle se `{}`, jinak co je v parametru, treba to `null`...